networking/k8s_nmstate/k8s-nmstate-observing-node-network-state.adoc

```diff
@@ -11,4 +11,5 @@ Node network state is the network configuration for all nodes in the cluster.
 
 include::modules/virt-viewing-network-state-of-node.adoc[leveloffset=+1]
 
-include::modules/virt-viewing-network-state-of-node-console.adoc[leveloffset=+1]
+include::modules/virt-viewing-graphical-representation-of-network-state-of-node-console.adoc[leveloffset=+1]
```

Comment:
Please add a blank line after each include statement (also applies to the other assembly).

source: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies